checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitSubchecker.java

```diff
@@ -45,6 +57,9 @@ protected Set<Class<? extends SourceChecker>> getImmediateSubcheckerClasses() {
     @Override
     public NavigableSet<String> getSuppressWarningsPrefixes() {
         NavigableSet<String> result = super.getSuppressWarningsPrefixes();
+        result.add("nullnessonly");
+        result.add("nullnesskeyfor");
+        result.add("nullnessinit");
         result.add("nullness");
```

Comment:
The "nullnessinit" is surprising, as this is the "NullnessNoInit" checker. Why is this needed?